oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,138 @@
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
+  Authoring and Catalog Course Authoring.
+
+  * This will depend on upcoming architectural infrastructure work on single sign on across the edX microservices and a universal
+    front end header and footer.
+
+* The following is Course Run metadata and its editing should be owned by the `Content Authoring subdomain`_:
+
+  * course run dates
+
+    * start
+    * end
+    * enrollment start
+    * enrollment end
+    * certificate availability
+
+  * pacing type
+
+* The following is Catalog Course metadata and its editing should be owned by the `Discovery subdomain`_:
+
+  * course about image
+  * instructors
+  * seat price
+  * seat type
+  * other catalog course marketing data
+
+  **Note:** course about image and instructors are currently managed as Course Run metadata but will transition to be Catalog Course metadata. 
+
+* The Discovery service will be the single source of truth for the published version of the above data. Though to address the data
+  synchronization issues, we will limit the editing of a piece of data to its single owning service.
+
+* We will introduce a new front end for editing Catalog Course metadata, written using the latest recommended front end technologies.
+
+  .. _Discovery subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _Content Authoring subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+  .. _edX subdomain: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+
+.. image:: oep-0027/publisher_v3_future.png
+  :alt: A flowchart of the movement of Course Run metadata and Catalog Course metadata through edX systems in the future.
+
+Consequences
+------------
+
+* By separating the concepts of Course Run metadata authoring from Catalog Course metadata authoring, we allow the overall
+  edX system to integrate better with other external systems. We have found that other systems use Open edX for the Course
+  Run learning experience and often choose to integrate with their own organizational Catalog Course management system.
+
+  As a consequence, the authoring experience is biforcated into two different subdomains in our system, Studio and Discovery. However,
+  we do hope that the integration at the UI layer will avoid any disparate user experience. The separation of concerns in the back end
+  allows us to have clearer integration points. We believe this is a worthwhile trade-off.
+
+* By removing the workflow engine from the system, we are reducing the turn around time for course provisioning. This introduces a potential
```

Comment:
I might say: "Be removing the workflow engine, we are moving to a model that enables self-service publishing for most courses and allows internal teams to reduce the number of blocking approvals and move to a more scaleable system of monitoring and alerting rather than manual review. There are risks associated with removing existing workflow and gates, but by offloading workflow to a specialized external system, there is a reduced reliance on engineering to make changes around workflow and ability to quickly make changes to workflow to respond to problems."